### PR TITLE
Add SQL Data Catalog - Collibra integration script

### PR DESCRIPTION
**Pull request by jameswrightrg**

### CODE REVIEW

File: SqlDataCatalog-Collibra/Collibra-Export.ps1 [hunk 1]
```diff
@@ -0,0 +1,42 @@
+$authToken = "[Your auth token]"
+$serverUrl = "http://[Your SQL Data Catalog Server FQDN]:15156" # or https:// if you've configured SSL
+$communityName = 'RedgateSqlDataCatalog'
+$synchronizationId = 'RedgateSqlDataCatalog'
+$collibraAPI = 'https://redgate.collibra.com/rest/2.0'
+$collibraUserName = "[collibra user name here]"
+$collibraPassword = "[collibra user password]"
+
+. ".\convert-to-collibra.ps1"
```
Comment (samilamti) [inline SqlDataCatalog-Collibra/Collibra-Export.ps1:9]:
```suggestion
. ".\ConvertTo-Collibra.ps1"
```
Let's use PowerShell nomenclature and casing whenever possible :)

File: SqlDataCatalog-Collibra/Collibra-Export.ps1 [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+. ".\collibra-api.ps1"
```
Comment (samilamti) [inline SqlDataCatalog-Collibra/Collibra-Export.ps1:10]:
```suggestion
. ".\Collibra-Api.ps1"
```

File: SqlDataCatalog-Collibra/Collibra-Export.ps1 [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+function export-database(
```
Comment (samilamti) [inline SqlDataCatalog-Collibra/Collibra-Export.ps1:12]:
```suggestion
function Export-Database(
```

File: SqlDataCatalog-Collibra/collibra-api.ps1 [hunk 4]
```diff
@@ -0,0 +1,84 @@
+
+#$Script:session = New-Object Microsoft.PowerShell.Commands.WebRequestSession
```
Comment (samilamti) [inline SqlDataCatalog-Collibra/collibra-api.ps1:2]: Is this something we want to keep in the sample?

File: SqlDataCatalog-Collibra/collibra-api.ps1 [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+    param (
+        [Parameter(Mandatory)][string] $jobId
+    )
+    $TimeEnd = (Get-Date).AddSeconds(60)
+
+    Do {
+        Start-Sleep -Seconds 1
+        $completedStates = @( "CANCELING", "COMPLETED", "CANCELED", "ERROR")
+
+        #https://redgate.collibra.com/rest/2.0/jobs/86e42720-23f6-462e-941d-486416e025d9
```
Comment (samilamti) [inline SqlDataCatalog-Collibra/collibra-api.ps1:59]: Is this something we want to keep in the sample?

File: SqlDataCatalog-Collibra/convert-to-collibra.ps1 [hunk 6]
```diff
@@ -0,0 +1,203 @@
...
+    $export += [pscustomobject]@{
+        resourceType = "Asset";
+        identifier   = $databaseIdentifier;
+        displayName  = $databaseName;
+        type         = [pscustomobject]@{
+            name = "Database"
+        };
+        status       = $acceptedStatus;
+        relations    = [pscustomobject]@{
+            "00000000-0000-0000-0000-000000007054:SOURCE" = @($instanceIdentifier)
```
Comment (samilamti) [inline SqlDataCatalog-Collibra/convert-to-collibra.ps1:100]: Is this a known constant value in Collibra, or do we need to make some sort of comment to change the value here?

File: SqlDataCatalog-Collibra/convert-to-collibra.ps1 [hunk 7]
```diff
@@ -0,0 +1,203 @@
...
+        $export += [pscustomobject]@{
+            resourceType = "Asset";
+            identifier   = $schemaIdentifier
+            displayName  = $schemaName;
+            type         = [pscustomobject]@{
+                name = "Schema"
+            };
+            status       = $acceptedStatus;
+            relations    = [pscustomobject]@{
+                "00000000-0000-0000-0000-000000007005:TARGET" = @($databaseIdentifier)
```
Comment (samilamti) [inline SqlDataCatalog-Collibra/convert-to-collibra.ps1:124]: Is this a known constant value in Collibra, or do we need to make some sort of comment to change the value here?

File: SqlDataCatalog-Collibra/convert-to-collibra.ps1 [hunk 8]
```diff
@@ -0,0 +1,203 @@
...
+            $export += [pscustomobject]@{
+                resourceType = "Asset";
+                identifier   = $tableIdentifier;
+                displayName  = $tableName;
+                type         = [pscustomobject]@{
+                    name = "Table"
+                };
+                status       = $acceptedStatus;
+                relations    = [pscustomobject]@{
+                    "00000000-0000-0000-0000-000000007043:SOURCE" = @($schemaIdentifier)
```
Comment (samilamti) [inline SqlDataCatalog-Collibra/convert-to-collibra.ps1:145]: Is this a known constant value in Collibra, or do we need to make some sort of comment to change the value here?

File: SqlDataCatalog-Collibra/convert-to-collibra.ps1 [hunk 9]
```diff
@@ -0,0 +1,203 @@
...
+                $export += [pscustomobject]@{
+                    resourceType = "Asset";
+                    identifier   = $columnIdentifier;
+                    displayName  = $columnName;
+                    type         = [pscustomobject]@{
+                        name = "Column"
+                    };
+                    status       = $acceptedStatus;
+                    relations    = [pscustomobject]@{
+                        "00000000-0000-0000-0000-000000007042:TARGET" = @($tableIdentifier)
```
Comment (samilamti) [inline SqlDataCatalog-Collibra/convert-to-collibra.ps1:190]: Is this a known constant value in Collibra, or do we need to make some sort of comment to change the value here?

File: SqlDataCatalog-Collibra/convert-to-collibra.ps1 (hunk 6 above)
Comment (jameswrightrg) [reply SqlDataCatalog-Collibra/convert-to-collibra.ps1]: This appears to be a Collibra constant. Unlike many of the other constants, we couldn't find a human-readable string to replace the GUID.

This goes for the other comments on this topic too.

**Pull request merged**